helix-core/src/indent.rs

```diff
@@ -896,6 +896,44 @@ pub fn get_scopes(syntax: Option<&Syntax>, text: RopeSlice, pos: usize) -> Vec<&
     scopes
 }
 
+pub fn get_highlight_name(
```

Comment:
I feel like this shouldn't be in this file. What would be a better file for this ? I put this in here initially because the treesitter-scopes function was in here